discovery-infra/start_discovery.py

```diff
@@ -166,6 +213,45 @@ def _cluster_create_params():
     return params
 
 
+def _find_free_cidr_machine(start_cidr_machine):
+    cidr_machine = IPNetwork(start_cidr_machine)
+    while _is_cidr_machine_exist(str(cidr_machine)):
+        cidr_machine += 1
+    return str(cidr_machine)
+
+
+def _is_cidr_machine_exist(cidr_machine):
+    cmd_output = utils.run_command(f'/usr/sbin/ip route show {cidr_machine}')
```

Comment:
it is ok but if you have 192.168.122.0/24 already used and user ask for 192.168.122.0/25 u will not fail but u can't really use it